### DISCUSSION TITLE
Container restart with error

### DISCUSSION THREAD

**Discussion by KallanX** [pg9182/northstar-dedicated#41] [Support] [answered]
Hello,
Not sure if this is the best place to post this but I'm currently running a T2 Northstar instance on a Ubuntu server. Running on Docker. For the most part, the system runs really well. But, at random times, the server will stop and restart. The only error I can find in the container logs is the following:

0024:err:virtual:virtual_setup_exception stack overflow 1808 bytes in thread 0024 addr 0x168c685a stack 0x208f0 (0x20000-0x21000-0x7d0000)

Any suggestions as to what this is or how to fix?

Server:
OS: Ubuntu 22.04.1 LTS x86_64
Kernel: 5.15.0-47-generic
CPU: 11th Gen Intel i7-1165G7 (4) @ 2.800GHz
Memory: 1595MiB / 3923MiB
Docker version 20.10.14, build a224086349

Docker compose file:
version: "3.9"

services:
  northstar1:
    image: ghcr.io/pg9182/northstar-dedicated:1-tf2.0.11.0-ns1.9.6
    pull_policy: always
    environment:
      - NS_PORT=37015
      - NS_PORT_AUTH=37115
      - 'NS_SERVER_NAME=[US-EAST] Saints of Virtue - 24/7 Attrition'
      - 'NS_SERVER_DESC=The official SoV Titanfall 2 server. Running on Linux w/Docker. Administrator: KallanX'
      - |
        NS_EXTRA_ARGUMENTS=
        +setplaylist aitdm
        +mp_gamemode aitdm
        +map mp_angel_city
        +ns_private_match_countdown_length 0
        +ns_should_return_to_lobby 0
        +net_compresspackets_minsize 64
        +net_compresspackets 1
        +spewlog_enable 0
        +sv_maxrate 127000
        +sv_balanceTeams 1
        +sv_cheats 0
    volumes:
      - /home/keith/tf2-files:/mnt/titanfall:ro
    ports:
      - '37015:37015/udp'
      - '37115:37115/tcp'
    restart: always

**Comment by Thunder1723** [comment, marked as answer]
I had the same error yesterday.  I was told that Northstar will occasionally crash so if it can successfully restart itself and continue plugging along it's just something you have to live with for now.
